README.md

```diff
@@ -62,71 +63,71 @@ Stripe is available for Store Owners and Merchants in:
 
 Please note, v4 of this gateway requires WooCommerce 3.0 and above.
 
-= Automatic installation =
+### Automatic installation
 
 Automatic installation is the easiest option as WordPress handles the file transfers itself and you don’t need to leave your web browser. To do an automatic install of the WooCommerce Stripe plugin, log in to your WordPress dashboard, navigate to the Plugins menu and click Add New.
 
 In the search field type “WooCommerce Stripe Payment Gateway” and click Search Plugins. Once you’ve found our plugin you can view details about it such as the point release, rating, and description. Most importantly, of course, you can install it by simply clicking "Install Now", then "Activate".
 
-= Manual installation =
+### Manual installation
 
 The manual installation method involves downloading our plugin and uploading it to your web server via your favorite FTP application. The WordPress codex contains [instructions on how to do this here](http://codex.wordpress.org/Managing_Plugins#Manual_Plugin_Installation).
 
-= Updating =
+## Updating
 
 Automatic updates should work like a charm; as always though, ensure you backup your site just in case.
 
-== Frequently Asked Questions ==
+## Frequently Asked Questions ##
 
-= Does this support recurring payments, like for subscriptions? =
+### Does this support recurring payments, like for subscriptions?
 
 Yes!
 
-= Does this require an SSL certificate? =
+### Does this require an SSL certificate?
 
 Yes! In Live Mode, an SSL certificate must be installed on your site to use Stripe. In addition to SSL encryption, Stripe provides an extra JavaScript method to secure card data using [Stripe Elements](https://stripe.com/elements).
 
-= Does this support both production mode and sandbox mode for testing? =
+### Does this support both production mode and sandbox mode for testing?
 
 Yes, it does - production and Test (sandbox) mode is driven by the API keys you use with a checkbox in the admin settings to toggle between both.
 
-= Where can I find documentation? =
+### Where can I find documentation?
 
 For help setting up and configuring, please refer to our [documentation](https://docs.woocommerce.com/document/stripe/).
 
-= Where can I get support or talk to other users? =
+### Where can I get support or talk to other users?
 
 If you get stuck, you can ask for help in the Plugin Forum.
 
-== Screenshots ==
+## Screenshots ##
 
 1. The Stripe payment gateway settings screen used to configure the main Stripe gateway.
 2. Offer a range of payment methods such as local and alternative payment methods.
 3. Pay with a saved payment method, a new card, and allow customers to save the payment card for future transactions.
 4. Apple Pay and other Payment Request buttons can be used on the Product Page and Checkout for express checkout.
 
-== Changelog ==
+## Changelog ##
 
-= 5.x.x - 2021-xx-xx =
+# 5.x.x - 2021-xx-xx #
```

Comment:
We should probably use a smaller heading here
![Screen Shot 2021-11-01 at 5 37 33 PM](https://user-images.githubusercontent.com/6216000/139751449-f4120a0f-2bb8-4dea-90d7-4721393f43b0.png)